src/axom/inlet/ConduitReader.hpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ *******************************************************************************
+ * \file ConduitReader.hpp
+ *
+ * \brief This file contains the class definition of the ConduitReader.
+ *******************************************************************************
+ */
+
+#ifndef INLET_CONDUITREADER_HPP
+#define INLET_CONDUITREADER_HPP
+
+#include "axom/inlet/Reader.hpp"
+
+#include "conduit.hpp"
+
+namespace axom
+{
+namespace inlet
+{
+/*!
+ *******************************************************************************
+ * \class ConduitReader
+ *
+ * \brief A Reader that is able to read variables from a file compatible with
```

Comment:
Can you add a message that the user is not to use this class directly but to use the yaml/json classes directly?